x/mint/keeper/hooks_test.go

```diff
@@ -201,38 +398,20 @@ func (suite *KeeperTestSuite) TestEndOfEpochNoDistributionWhenIsNotYetStartTime(
 //
 // Ref: https://github.com/osmosis-labs/osmosis/issues/1917
 func (suite *KeeperTestSuite) TestAfterEpochEnd_FirstYearThirdening_RealParameters() {
-	// Most values in this test are taken from mainnet genesis to mimic real-world behavior:
```

Comment:
Note to reviewers: these have been moved to the top of the file to be shared between tests